SwiftTwitter/Scenes/Login/LoginViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  LoginViewController.swift
+//  SwiftTwitter
+//
+//  Created by Matheus Lima on 02/05/19.
+//  Copyright (c) 2019 Matheus Lima. All rights reserved.
+//
+//  This file was generated by the Clean Swift Xcode Templates so
+//  you can apply clean architecture to your iOS and Mac projects,
+//  see http://clean-swift.com
+//
+
+import UIKit
+
+protocol LoginDisplayLogic: class {
+    func displayLoginSuccess(viewModel: Login.OAuth.ViewModel)
+    func displayError(viewModel: Login.OAuth.ViewModel)
+}
+
+class LoginViewController: UIViewController {
+    var interactor: LoginBusinessLogic?
+    var router: (NSObjectProtocol & LoginRoutingLogic & LoginDataPassing)?
+    
+    // MARK: - Object lifecycle
+    
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+        self.setup()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.setup()
+    }
+    
+    // MARK: - Setup
+    
+    private func setup() {
+        let viewController = self
+        let interactor = LoginInteractor()
+        let presenter = LoginPresenter()
+        let router = LoginRouter()
+        viewController.interactor = interactor
+        viewController.router = router
+        interactor.presenter = presenter
+        presenter.viewController = viewController
+        router.viewController = viewController
+        router.dataStore = interactor
+    }
+    
+    // MARK: - Routing
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if let scene = segue.identifier {
+            let selector = NSSelectorFromString("routeTo\(scene)WithSegue:")
+            if let router = self.router, router.responds(to: selector) {
+                router.perform(selector, with: segue)
+            }
+        }
+    }
+    
+    // MARK: - Instanciation
+    
+    class func instanceFromStoryboard() -> LoginViewController {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        return storyboard.instantiateViewController(withIdentifier: "LoginViewController") as! LoginViewController
+    }
+
+    // MARK: - View lifecycle
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+    
+    @IBAction func loginAction(_ sender: Any) {
+        self.login()
+    }
+    
+    func login() {
+        let request = Login.OAuth.Request()
+        self.interactor?.login(request: request)
+    }
+}
+
+extension LoginViewController: LoginDisplayLogic {
+    func displayError(viewModel: Login.OAuth.ViewModel) {
+        guard let errorMessage = viewModel.errorMessage else { return }
```

Comment:
we always put an endline after opening and before closing brackets.
else {
 return 
}